ready_recipes/conda-testenv/meta.yaml

```diff
@@ -32,3 +32,8 @@ about:
   home: https://github.com/SciTools/conda-testenv
   license: BSD
   summary: 'Run tests of all packages in an environment.'
+
+extra:
+  recipe-maintainers:
+    - lbdreyer
```

Comment:
Ping @lbdreyer. I just wanted you to know about this one. Once merged, you'll receive an email inviting you into the conda-testenv-maintainers team. Hope that's ok.